openjdk.test.load/src/test.load/net/adoptopenjdk/stf/MauveMultiThreadTestSpcl.java

```diff
@@ -25,7 +25,7 @@
 import net.adoptopenjdk.stf.processes.definitions.LoadTestProcessDefinition;
 import net.adoptopenjdk.stf.runner.modes.HelpTextGenerator;
 
-public class MauveMultiThreadLoadTestWithTraceOption implements StfPluginInterface {
+public class MauveMultiThreadTestSpcl implements StfPluginInterface {
```

Comment:
Instead of Spc (which stands for spec?) how about Trc (for Trace).

Reply:
Makes sense. Updated the name. (New test: Grinder/8560/console)